DESCRIPTION

```diff
@@ -40,7 +40,7 @@ Depends:
     R (>= 4.0)
 Imports:
     admiraldev (>= 0.5.0.9000),
-    dplyr (>= 0.8.4),
+    dplyr (>= 1.0.5),
```

Comment:
ummm...so this update removes capability with dplyr less than 1.0.5?  Maybe we should say that in our news now??

Any chance we could still work with that version or "are the times they is changing"??

Reply:
I actually just grabbed it from the `DESCRIPTION` file of `admiraldev` but this has technically been true for a while because we use functions like `across()` which was introduced at `1.0.0` and gradually improved

Reply:
well then... i guess we just continue on with this update